static/app/views/performance/transactionSummary/transactionThresholdModal.spec.tsx

```diff
@@ -1,38 +1,28 @@
-import {mountWithTheme} from 'sentry-test/enzyme';
-import {act} from 'sentry-test/reactTestingLibrary';
-import {selectByLabel} from 'sentry-test/select-new';
+import selectEvent from 'react-select-event';
 
+import {act, render, screen, userEvent, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
 import ProjectsStore from 'sentry/stores/projectsStore';
 import EventView from 'sentry/utils/discover/eventView';
-import TransactionThresholdModal from 'sentry/views/performance/transactionSummary/transactionThresholdModal';
+import TransactionThresholdModal, {
+  TransactionThresholdMetric,
+} from 'sentry/views/performance/transactionSummary/transactionThresholdModal';
 
 const stubEl = props => <div>{props.children}</div>;
 
-function clickSubmit(wrapper) {
-  // Click on submit.
-  const button = wrapper.find('Button[data-test-id="apply-threshold"] button');
-  button.simulate('click');
-  return tick();
-}
-
-function clickReset(wrapper) {
-  // Click on submit.
-  const button = wrapper.find('Button[data-test-id="reset-all"] button');
-  button.simulate('click');
-  return tick();
-}
-
 function mountModal(eventView, organization, onApply) {
```

Comment:
can we add types here?

```suggestion
function mountModal(
  eventView: EventView,
  organization: Organization,
  onApply: React.ComponentProps<typeof TransactionThresholdModal>['onApply']
) {
```